src/Microsoft.ML.AutoML.Interactive/Microsoft.ML.AutoML.Interactive.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
Should it be `true` instead?

Reply:
No, it's false because we are going the same route as the DataFrame where this is bundled as part of the AutoML package so this one shouldn't be packed by itself.